### ISSUE TITLE
Allow specification of input date/time

### ISSUE THREAD

**Issue by TotallyInformation** [TotallyInformation/node-red-contrib-moment#15]
Via request from [Björn Bellmann in the Google Group](https://groups.google.com/forum/#!topic/node-red/16RThDkQkpg).

It would be good to add an extra, optional input parameter that specified the input date/time format,

**Comment by Rene-Auberger** [comment]
This is needed for example to parse the awkward format returned by the Twitter API which for example returns created_at = "Mon Jan 08 21:24:37 +0000 2018" which is "dd MMM DD HH:mm:ss ZZ YYYY"

**Comment by shrickus** [comment]
Julian, I just submitted a pull request for a partial fix that will cover the Twitter API use case...

Basically, whenever the input string format is not recognized, I added some fallback logic that tries the create a new Date from the raw string. At least these two (previously failing) inputs will now be accepted:
```
> new Date("Mon Jan 08 21:24:37 +0000 2018")  // twitter api format
2018-01-08T21:24:37.000Z
> new Date("2018-01-08 21:24:37.000000000 -0500")  // file stat format
2018-01-09T02:24:37.000Z
```

**Comment by TotallyInformation** [comment]
Thanks Steve, this has been accepted and published as v1.0.7

**Comment by TotallyInformation** [comment]
Added to backlog for a future release, closing for now.
